Self service receipt import script/README.md

```diff
@@ -0,0 +1,22 @@
+# A Node script that parses a CSV file of fetch tokens and imports them into [RevenueCat](https://www.revenuecat.com) using the REST API.
+**License: MIT**
+
+If you have at least 100k in monthly revenue, you can [reach out](https://www.revenuecat.com/demo/) to perform an assisted bulk receipt import.
+
+Not sure if this self-service script is right for you? [Get in touch](https://app.revenuecat.com/settings/support)!
+
+## ⚠️ Warning!
+This script is automated and performs changes to the data in RevenueCat that may be difficult to reverse. If you're not a developer, have one help you. If you are a developer, test the script on a small subset of the CSV file and check how it looks in RevenueCat before running it on the full file.
```

Comment:
This was missing in the original, I added this in today

Reply:
Great warning 🙌